anago

```diff
@@ -211,7 +211,9 @@ check_prerequisites () {
       return 1
     fi
   done
+
   # Ensure $USER is active to start
+  gcloud_auth_list=$($GCLOUD auth list 2>/dev/null)
```

Comment:
I just realized that the behavior in the previous loop is pretty subtle. It might be worth adding a comment there that you intentionally have $userat last so that it ends up as the set account.